piptools/repositories/base.py

```diff
@@ -30,3 +30,11 @@ def get_dependencies(self, ireq):
         dependencies (also InstallRequirements, but not necessarily pinned).
         They indicate the secondary dependencies for the given requirement.
         """
+
+    @abstractmethod
+    def get_hashes(self, ireq):
```

Comment:
This method needs to be added to the `tests.conftest.FakeRepository` as well or else the tests fail